composer.lock

```diff
@@ -366,20 +367,20 @@
         },
         {
             "name": "pear-pear.horde.org/Horde_Nls",
-            "version": "2.1.0",
+            "version": "2.0.3",
```

Comment:
I don't get this. Even if a specify ``^2.1.0@stable`` in composer.json, which is the lastest version [here](http://pear.horde.org/), 2.0.3 is installed :-/